ts/Dashboards/Board.ts

```diff
@@ -583,6 +584,30 @@ class Board implements Serializable<Board, Board.JSON> {
 
         return options;
     }
+
+    /**
+     * Get a Dashboards component by its id.
+     *
```

Comment:
Missing `@param` for the id?

Reply:
@jomar-honsi seems that is still missing